schemaless/create_uuid_map.py

```diff
@@ -253,8 +392,51 @@ def build(self):
 
         # Resolve all parent UUIDs and generate new UUIDs
         rg._assign_uuids()
+
+        if not self.find_likely_matches:
+            return rg
+
+        # Now find likely matches
+        for source_name, source_records in latest_records.items():
+            for fk, record in source_records.items():
+                likely_parents = []
+                likely_children = []
+                self.helpers[source_name].process_likely(
+                    fk, record, likely_parents, likely_children)
+
+                if self.exclude_known_likely_matches:
+                    # Now that we have all likely parents, remove parents
+                    # with an explicit or implied link
+                    my_uuid = rg.get(fk).uuid
+                    parents = []
+                    children = []
+                    for parent in likely_parents:
+                        if rg.get(parent).uuid == my_uuid:
+                            continue
+                        parents.append(parent)
+                else:
+                    parents = likely_parents
+                    children = likely_children
+                self.likelies[fk] = {
+                    'parents': parents,
+                    'children': children,
+                }
+
         return rg
 
+    def write_likely_matches(self, outfile):
+        with open(outfile, 'w') as lf:
+            writer = DictWriter(
+                lf, fieldnames=['record_id', 'parents', 'children'])
+            writer.writeheader()
+            for fk, rec in self.likelies.items():
+                if rec['parents'] or rec['children']:
+                    writer.writerow({
+                        'record_id': fk,
```

Comment:
Just a thought -- our fks may be potentially confusing if we're trying to provide the output of this to folks in the city. Not sure the best case to handle that, may documentation that goes with the spreadsheet or something

Reply:
Ah, good point. Should be a simple fix to handle this.

Reply:
Actually, there's not a very clean way that I can see to do this. Opening this as a bug to address if Svetha can't work with what we give her: #103